unfetter-discover/unfetter-ui
# push state to url, remember state on refresh, css improvements

supports unfetter-discover/unfetter#644

# To Test
* `git checkout issue-644/results-view`
* visit an assessment result view
* click the phases, notice the url changes
* refresh the url, notice the correct phase state is displayed
* note, attack pattern state will be handled soon as well
* `npm run test` verify
